src/views/chainState/index.tsx

```diff
@@ -79,6 +84,29 @@ const ChainState = () => {
     }
   }, [palletsWithStorage]);
 
+  useEffect(() => {
+    if (palletSelected?.name && storageSelected?.name && dynamicBulder) {
+      try {
+        const storageCodec = dynamicBulder.buildStorage(palletSelected.name, storageSelected.name);
+
+        const encodedKey = storageCodec.enc(...([callArgs].filter(arg => Boolean(arg))));
+        setEncodedStorageKey(encodedKey);
+
+        // const decodedArgs = storageCodec.keyDecoder(encodedKey);
```

Comment:
Let's either leave comment in case we want to persist the commented lines or remove them

Reply:
removed